src/implementations/osmosis.rs

```diff
@@ -105,18 +106,7 @@ impl Mint for OsmosisDenom {
             attr("recipient", recipient.to_string()),
         ]);
 
-        Ok(Response::new()
-            .add_messages(vec![
-                mint_msg,
-                CosmosMsg::Bank(BankMsg::Send {
-                    to_address: recipient.to_string(),
-                    amount: vec![Coin {
-                        denom: self.to_string(),
-                        amount,
-                    }],
-                }),
```

Comment:
not that this matters now. but can you send coins to yourself or is that an invalid message?

Reply:
You mean to create a BankMsg with recipient yourself? Should work. I've done it before just to put something in the memo field